pkgs/applications/networking/pyload/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv, fetchFromGitHub, fetchurl, python, gocr, unrar, rhino, spidermonkey }:
+stdenv.mkDerivation rec {
+  version = "stable";
+  name = "pyLoad-" + version;
+
+  src = fetchFromGitHub {
+    owner = "pyload";
+    repo = "pyload";
+    rev = "stable";
+    sha256 = "1n1l3cznkaqga351asv1998qsmlnlbbcinm8dyjy5rwcs3k590vc";
+  };
+
+  installPhase =
+    let
+      pyLoadBin = fn: ''
+        #!/usr/bin/env bash
+        ${pythonWithPkgs}/bin/python $out/${fn}.py \$@
+      '';
+      pyLoadCore = pyLoadBin "pyLoadCore";
+      pyLoadCli = pyLoadBin "pyLoadCli";
+    in ''
+      mkdir $out
+      mkdir $out/bin
+
+      cp -r . $out/
+
+      echo "${pyLoadCore}" > $out/bin/pyLoadCore
+
+      echo "${pyLoadCli}" > $out/bin/pyLoadCli
+
+      chmod +x $out/bin/*
+
+    '';
+
+  patches =
+    # gets merged in next release version of pyload
+    let configParserPatch = fetchurl {
+      url = "https://patch-diff.githubusercontent.com/raw/pyload/pyload/pull/2625.patch";
+      sha256 = "0v8l840yxgzlbcxnm7cd4ipc52bbs798lnbyd6f9nrd85ih1nsfl";
+    };
+    in [configParserPatch];
+
+  pythonWithPkgs = (python.withPackages (ps: with ps; [
```

Comment:
this needs to be in a let expression instead of passing it as an argument to mkDerivation